ci/scripts/assume-role.js

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const AWS = require("aws-sdk")
+
+const sts = new AWS.STS()
+
+const run = async function run() {
+  const assumeRoleResponse = await sts.assumeRole({
+    RoleArn: process.env.AWS_ROLE_ARN,
+    RoleSessionName: process.env.AWS_ROLE_SESSION_NAME
+  }).promise()
+
+  const {
+    AccessKeyId: AWS_ACCESS_KEY_ID,
+    SecretAccessKey: AWS_SECRET_ACCESS_KEY,
+    SessionToken: AWS_SESSION_TOKEN
+  } = assumeRoleResponse.Credentials
+
+  fs.writeFileSync('assume-role/assume-role.yml', JSON.stringify({ AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, AWS_SESSION_TOKEN }))
```

Comment:
It's a bit confusing we're writing json to a yml file. `assume-role.json` perhaps?

Reply:
yeah I was just copying the previous thing. I'll change it